portality/formcontext/forms.py

```diff
@@ -116,9 +116,9 @@ class JournalInformation(Form):
     )
     digital_archiving_policy_url = URLField('Enter the URL where this information can be found',
         [OptionalIf('digital_archiving_policy', optvals=Choices.digital_archiving_policy_url_optional()), URLOptionalScheme()],
-        description='This field is optional if you have only selected "No policy in place" above',
+        description='This field is optional if you selected "No policy in place".',
     )
-    crawl_permission = RadioField('Does the journal allow anyone to crawl the full-text of the journal?',
+    crawl_permission = RadioField('Does the journal allow software/spiders to automatically crawl the journal content (also known as text mining)?',
```

Comment:
Do we still call them spiders? I'd say crawler.

As a side note, I had a hilarious time googling this. Spider... in a web context. "spider web"? ha, nope. "spider internet" does give the definition you mean, but I still think crawler is better language for it.